keps/349-all-or-nothing/README.md

```diff
@@ -315,9 +331,11 @@ type RequeueState struct {
 
 We introduce a new workload condition, called `PodsReady`, to indicate
 if the workload's startup requirements are satisfied. More precisely, we add
-the condition when `job.status.ready + job.status.succeeded` is greater or equal
+the condition when `job.status.ready + len(job.status.uncountedTerimnatedPods.succeeded) + job.status.succeeded` is greater or equal
 than `job.spec.parallelism`.
 
+Note that we count `job.status.uncountedTerminatedPods` - this is meant to prevent flickering of the `PodsReady` condition when pods are transitioning to the `Succeeded` state.
```

Comment:
Does this improve the existing waitForPodsReady feature even if we do not introduce the recoveryTimeout?

Reply:
Yes